CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Thank you for your interest in contributing to WorldEdit! We appreciate your
 effort, but to make sure that the inclusion of your patch is a smooth process, we
 ask that you make note of the following guidelines.
 
-* **Follow the [Oracle coding conventions](http://www.oracle.com/technetwork/java/codeconv-138413.html).**
+* **Follow the [Oracle coding conventions](https://www.oracle.com/java/technologies/cc-java-programming-language.html).**
```

Comment:
I wonder if we should just remove this, @wizjany @me4502 thoughts?

Reply:
Tbf, I only bothered with the `rebase` link, for this one I had to go back to early 2014 to find a suitable redirect.

Reply:
Yeah I'm into just removing this. Do we even follow these conventions? I swear it's more Google-based